src/index.css

```diff
@@ -9,3 +9,22 @@ body {
 .left-spinner {
   margin: 10px 10px;
 }
+
```

Comment:
Isn't it supposed to go in component scss?

Reply:
The .left-spinner was there before :-) and that makes me think to add it here as first place.
Yes, it is better to move to a close scss file at same component/page level.

Reply:
yup sorry, my comment was for lines _below_